docs/installguide/install_all.rst

```diff
@@ -7,6 +7,13 @@ Windows
 Upgrading KA Lite over an existing installation is easy -- just run the installer and follow the prompts!
 You don't need to uninstall your old KA Lite installation first.
 
+Upgrade
+_______
+
+To upgrade KA Lite on Windows, simply download the latest installer and run it.
+You can follow the prompts to either keep your existing data or delete old data and start over.

```

Comment:
I don't think this is true in all cases. It's true if you run `kalite manage setup` because it's interactive. But I think that the case of the deb is that it'll restart the system service, and the process will be non-interactive. In case the user chose to download assessment items during the initial installation, those assessment items will be re-downloaded and the original ones will be deleted and replaced.